mix/Extension.cpp

```diff
@@ -17,13 +17,14 @@
  * Ethereum IDE client.
  */
 
+#include "AppContext.h"
```

Comment:
best to include just the required boost includes there then, rather than all the other stuff that AppContext brings in.